sys-apps/sum/sum-2.0.1.20180420.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
How would you feel about renaming it to `sys-apps/smc-sum` appropriately? I think having matching executable and package name would be less confusing to users, and you can create `MY_P` to replace two instances of current `${PN}_${MY_PV}`.

Reply:
Fine for me, changed!